java/src/jmri/jmrix/mqtt/Bundle.properties

```diff
@@ -28,6 +28,9 @@ NameTopicSignalHead=SignalHead receive topic:
 
 NameTopicSignalMast=SignalMast receive topic:
 
+NameTopicLastWill=Last will topic
```

Comment:
Minor point (or two):  should lines 31 and 32 end in colon (:) for consistency with other topic description text that will appear on screen?

Also, is line 50 (TopicLightCommand) superfluous?  It does not appear to be used in MqttAdapter.java as are the other Topic[..]?  If it is for future use, perhaps that could be commented?